### DISCUSSION TITLE
Feature Request: Date Field Default Date

### DISCUSSION THREAD

**Discussion by stephenmoore33** [docusealco/docuseal#50] [Polls]
Great product!

Please allow the option to default the date to 'Today'

**Comment by AlexBTurchyn** [comment]
@stephenmoore33 This is quite a controversial improvement because it is very important that the signers enter the data themselves. Otherwise, they may later say that they were misled

**Reply by stephenmoore33** [reply]
Great point! What about an additional field of 'Current Date Stamp' that dates and times the document at signing/submission?

**Reply by AlexBTurchyn** [reply]
I like this idea better https://github.com/docusealco/docuseal/discussions/42
